multiple definition of `accepted_macs'

**Describe the bug**
```
mkdir -p build
Run 'sudo make install' for installation.
Run 'sudo make uninstall' for uninstallation.
cd src && make
make[1]: Entering directory '/home/yochanan/Documents/AUR/linux-wifi-hotspot/src/linux-wifi-hotspot-3.2.0/src'
/usr/bin/glib-compile-resources ui/glade/wifih.gresource.xml --target=ui/resources.c --sourcedir=ui/glade --generate-source
make[2]: Entering directory '/home/yochanan/Documents/AUR/linux-wifi-hotspot/src/linux-wifi-hotspot-3.2.0/src'
gcc -c -o ../build/main.o ui/main.c `pkg-config --cflags gtk+-3.0`
gcc -c -o ../build/ui.o ui/ui.c `pkg-config --cflags gtk+-3.0`
gcc -c -o ../build/h_prop.o ui/h_prop.c `pkg-config --cflags gtk+-3.0`
gcc -c -o ../build/util.o ui/util.c `pkg-config --cflags gtk+-3.0`
g++ -c -o ../build/read_config.o ui/read_config.cpp
gcc -c -o ../build/resources.o ui/resources.c `pkg-config --cflags gtk+-3.0`
gcc -o ../build/wihotspot-gui ../build/main.o ../build/ui.o ../build/h_prop.o ../build/util.o ../build/read_config.o ../build/resources.o `pkg-config --cflags gtk+-3.0` `pkg-config --libs gtk+-3.0 --libs x11` -lstdc++
/usr/sbin/ld: ../build/h_prop.o:(.bss+0x2000): multiple definition of `accepted_macs'; ../build/ui.o:(.bss+0x128): first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:42: wihotspot-gui] Error 1
make[2]: Leaving directory '/home/yochanan/Documents/AUR/linux-wifi-hotspot/src/linux-wifi-hotspot-3.2.0/src'
make[1]: *** [Makefile:33: resources.c] Error 2
make[1]: Leaving directory '/home/yochanan/Documents/AUR/linux-wifi-hotspot/src/linux-wifi-hotspot-3.2.0/src'
make: *** [Makefile:5: all] Error 2
```

**To Reproduce**
Build 3.2.0 via AUR PKGBUILD

**Desktop**
 - Manjaro GNOME 3.38.1

--- a/src/ui/h_prop.c
+++ b/src/ui/h_prop.c
@@ -57,7 +57,7 @@ char cmd_write_mac[BUFSIZE];
 char h_running_info[BUFSIZE];
 char interface_list[BUFSIZE];
 char wifi_interface_list[BUFSIZE];
-char accepted_macs[BUFSIZE];
+char _accepted_macs[BUFSIZE];
 
 const char* g_ssid=NULL;
 const char* g_pass=NULL;
@@ -177,10 +177,10 @@ char * read_mac_filter_file(char * filename){
     }
 
     while((ch = (char)fgetc(fp)) != EOF)
-        strcat(accepted_macs, &ch);
+        strcat(_accepted_macs, &ch);
 
    fclose(fp);
-   return accepted_macs;
+   return _accepted_macs;
 }
 
 //int write_config(char* file){

--- a/src/ui/h_prop.c
+++ b/src/ui/h_prop.c
@@ -48,19 +48,19 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define STOP "--stop"
 
 
-char cmd_start[BUFSIZE];
-char cmd_mkconfig[BUFSIZE];
-char cmd_config[BUFSIZE];
-char cmd_kill[BUFSIZE];
-char cmd_write_mac[BUFSIZE];
+static char cmd_start[BUFSIZE];
+static char cmd_mkconfig[BUFSIZE];
+static char cmd_config[BUFSIZE];
+static char cmd_kill[BUFSIZE];
+static char cmd_write_mac[BUFSIZE];
 
-char h_running_info[BUFSIZE];
-char interface_list[BUFSIZE];
-char wifi_interface_list[BUFSIZE];
-char _accepted_macs[BUFSIZE];
+static char h_running_info[BUFSIZE];
+static char interface_list[BUFSIZE];
+static char wifi_interface_list[BUFSIZE];
+static char accepted_macs[BUFSIZE];
 
-const char* g_ssid=NULL;
-const char* g_pass=NULL;
+static const char* g_ssid=NULL;
+static const char* g_pass=NULL;
 
 //config_t cfg;
 
@@ -177,10 +177,10 @@ char * read_mac_filter_file(char * filename){
     }
 
     while((ch = (char)fgetc(fp)) != EOF)
-        strcat(_accepted_macs, &ch);
+        strcat(accepted_macs, &ch);
 
    fclose(fp);
-   return _accepted_macs;
+   return accepted_macs;
 }
 
 //int write_config(char* file){

